pkg/api/azenvtypes.go

```diff
@@ -190,12 +191,13 @@ var (
 			ResourceManagerVMDNSSuffix: "cloudapp.microsoftazure.de",
 		},
 		OSImageConfig: map[Distro]AzureOSImageConfig{
-			Ubuntu:     DefaultUbuntuImageConfig,
+			Ubuntu:     Ubuntu1604ImageConfig,
 			Ubuntu1804: Ubuntu1804ImageConfig,
-			RHEL:       DefaultRHELOSImageConfig,
-			CoreOS:     DefaultCoreOSImageConfig,
-			AKS:        DefaultUbuntuImageConfig,
-			AKS1804:    Ubuntu1804ImageConfig,
+			RHEL:       RHELOSImageConfig,
+			CoreOS:     CoreOSImageConfig,
+			AKS:        Ubuntu1604ImageConfig,
+			AKS1604:    Ubuntu1604ImageConfig,
+			AKS1804:    Ubuntu1604ImageConfig, // workaround for https://github.com/Azure/aks-engine/issues/761
```

Comment:
this is an ugly hack to prevent Sovereign cloud users to run into #761 by default... Normally, this should have been `Ubuntu1804ImageConfig` but the `ubuntu-18.04` image causes OS provisioning timeouts so we keep using `ubuntu-16.04` for US Gov and German Cloud.